### ISSUE TITLE
Doesnt work with replays of ballchasing.com

### ISSUE THREAD

**Issue by Lukas2380** [MrPh1l/ReplayMapChanger#2]
When trying to convert a replay of ballchasing.com to a different map. I get this message and it does not work. Neither when downloading the replay nor when you watch the replay with this plugin for watching replays of ballchasing.com:
https://bakkesplugins.com/plugins/view/214

![image](https://user-images.githubusercontent.com/96386473/227722705-5f07d016-1106-4aea-9935-fe61237199b1.png)

**Comment by CosmoticRL** [comment]
Did you fix it? I’m having the same issue

**Comment by Lukas2380** [comment]
Nope. But if since you are asking me like that, you should look into the ballchasing replay viewer plugin. 
I think what happens is that when viewing a replay with the replay player, the plugin saves the replay to some file location, probably in the bakkesmod/replayviewer folder. 
Maybe: 
Then your plugin cant find the replay?
or
It finds the replay but the replayviewer plugin deletes the replay file immediately after you are trying to manipulate the file.

The problem doesnt bother me anymore since I just needed it for making a montage. If you dont want to fix that problem you could at least acknoledge it in the readme, so people with the same issue dont have to message you about it.
The easiest solution is to just download the file instead of using the replay viewer plugin.

-Edit: Ive never commented on a github issue before so ignore that I thought you were the creator of this plugin xD

**Comment by MrPh1l** [comment]
This issue has now been fixed with v1.0.1.
